blackbox/docs/admin/auth/hba.txt

```diff
@@ -160,3 +159,33 @@ the the ``auth.host_based`` setting, like this:
         config:
           0:
             user: crate
+
+Authenticating to Admin UI
+==========================
+
+.. hide:
+
+    cr> CREATE USER admin;
+    CREATE OK, 1 row affected (... sec)
+
+When trying to access the Admin UI of CrateDB, initially an authentication with
+the user defined with the
+:ref:`auth.trust.http_default_user <auth_trust_http_default_user>` setting
+(defaults to ``crate``) will be attempted. If this authentication attempt
+fails, then the browser will open the standard popup window where the user is
+asked to fill in credentials. Depending on the HBA configuration it maybe not
+be necessary to provide a password but only a username. Users that log in to
+the Admin UI must be granted `DQL`` privileges at ``CLUSTER`` level in order to
+be able to access the various monitoring sections. For example::
+
+    cr> GRANT DQL TO admin;
```

Comment:
I think it would be fine to just link to the DQL reference instead of inlining an example.

Reply:
We've discussed that and I was thinking to have some reference to the guide section where it's more tutorial like but we agreed it's nice for the moment being to have the example here. (it's very short anyways)

Reply:
I thought we already have this - or at least it's planned/being worked on, but I guess @nslater can provide more input here.

Reply:
we don't have anything in the guide yet. for now, an example here is good